src/docs/concepts/distribution-summaries.adoc

```diff
@@ -23,6 +23,11 @@ DistributionSummary summary = DistributionSummary
 1. Add base units for maximum portability -- base units are part of the naming convention for some monitoring systems. Leaving it off and violating the naming convention will have no adverse effect if you forget.
 2. Optionally, you may provide a scaling factor that each recorded sample will be multipled by as it is recorded.
 
+NOTE: Max for basic `DistributionSummary` implementations such as `CumulativeDistributionSummary`, `StepDistributionSummary` is a time window max (`TimeWindowMax`).
+It means that its value is the maximum value during a time window.
+If the time window ends, it'll be reset to 0 and a new time window starts again.
```

Comment:
It's a rolling time window, so if there are new values recorded before the time window length, then the max won't be 0. It will only be 0 in the case of no new values (or all new values being 0).
```suggestion
If no new values are recorded for the time window length, the max will be reset to 0 as a new time window starts.
```

Reply:
@shakuzen Thanks! I applied the suggested change, and I also applied it to the change for `Timer`.